docs/source-pytorch/extensions/callbacks.rst

```diff
@@ -138,6 +138,11 @@ Callback API
 ************
 Here is the full API of methods available in the Callback base class.
 
+.. note::
+
+    Outputs: Hooks that occur on batch end (e.g. ``on_train_batch_end``) will receive any values returned from that step (such as the loss) in the form of a dictionary named ``outputs``.
+    In the case ``accumulate_grad_batches`` > 1, these values will be normalized by Lightning so that the value provided in ``outputs`` will be the value returned from the training/validation/predict step divided by ``accumulate_grad_batches``
```

Comment:
1. only loss is scaled... Other outputs are kept as it is.
2. Also it's the case only with `training_step`. For rest it is same.

I'd recommend adding it to the `training_step` docstring too.

Reply:
> 1. only loss is scaled... Other outputs are kept as it is.
> 2. Also it's the case only with `training_step`. For rest it is same.
> 
> I'd recommend adding it to the `training_step` docstring too.

Thanks, I'll fix and submit. By `training_step` do you mean the `on_train_batch_end` hook? Or do you want me adjust the actual `training_step` docstring?

Reply:
adjust actual training_step hook. just add a note that the loss returned from here will be normalized internally.

Reply:
Updated. I tried to find the best location in the training step docstring, and I updated the callback docs to include the specific details you mentioned. Please let me know if further fixes are needed and thank you for the time and help.